custom/auth/ante/min_initial_deposit.go

```diff
@@ -0,0 +1,77 @@
+package ante
+
+import (
+	"fmt"
+	//"strconv"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	core "github.com/terra-money/core/types"
+)
+
+// MinInitialDeposit Decorator will check Initial Deposits for MsgSubmitProposal
+type MinInitialDepositDecorator struct {
+	govKeeper GovKeeper
+	treasuryKeeper TreasuryKeeper
+}
+
+// NewMinInitialDeposit returns new min initial deposit decorator instance
+func NewMinInitialDepositDecorator(govKeeper GovKeeper, treasuryKeeper TreasuryKeeper) MinInitialDepositDecorator {
+	return MinInitialDepositDecorator{
+		govKeeper: govKeeper,
+		treasuryKeeper: treasuryKeeper,
+	}
+}
+
+// IsMsgSubmitProposal checks wheter the input msg is a MsgSubmitProposal
```

Comment:
Looks like the linter also wants part of the action :P

Reply:
Linter is happy